### PR TITLE
Add InputEmailChip component

### LINKED ISSUE #560: Create component for an input with inline email chips
Figma: https://www.figma.com/file/0U2YaGkfYfn8SAWrrLzEjW/neetoDesk?node-id=1411%3A13373&t=lOdJWaClWFyIf5w1-1

image

### PR DESCRIPTION
**Pull request by chiragbhaiji**
Fixes: #560 #557 

@sangameshsomawar please review

**Checklist**
- [x] I have performed a self-review of my code and tested it well before raising the PR.
- [x] I have made corresponding changes to the documentation and `index.d.ts`.
- [x] I've verified the change via .yalc in neetoDesk.  

**Demo or Screenshots**

Demo link: https://app.usebubbles.com/vJnXyiKBRMoGM1qF34jpMA/input-email-chip-component-demo

### CODE REVIEW

File: src/components/InputEmailChip.jsx
Comment (sangameshsomawar) [inline src/components/InputEmailChip.jsx:30]: @chiragbhaiji 

https://app.usebubbles.com/cwEjgqgxQknywBJD8uwC1X/sangamesh-s-video-recording

**Review by sangameshsomawar — CHANGES_REQUESTED** [review]

File: storybook/stories/Chip.stories.jsx
Comment (chiragbhaiji) [reply storybook/stories/Chip.stories.jsx:21]: Sorry I was testing the case of long strings after making updates to the Chip component. Will revert it.

**Comment by chiragbhaiji** [comment]
@sangameshsomawar _a I've updated and fixed things based on the provided feedback. Please review the changes and check [this issue](https://app.usebubbles.com/rfPwpb8ckqfA9P7fVNAu1e/chirag-s-video-recording) as it requires your input for the fix.

**Review by sangameshsomawar — APPROVED** [review]
@chiragbhaiji _a LGTM, please test once and merge.

**Comment by chiragbhaiji** [comment]
> @chiragbhaiji _a LGTM, please test once and merge.

Tested it on Pixel 4a emulator and it looks good on that too. So merged it.

**Pull request merged**
